pkg/releasetesting/test_suite_test.go

```diff
@@ -37,6 +37,29 @@ import (
 	tillerEnv "k8s.io/helm/pkg/tiller/environment"
 )
 
+const manifestWithTestHook = `
```

Comment:
should we add a manifest for the new `test-failure` hook and associated test(s)?